PHP/CSS (for PHP).sublime-syntax

```diff
@@ -0,0 +1,38 @@
+%YAML 1.2
+---
+name: CSS (for PHP)
+hidden: true
+scope: source.css.php
+version: 2
+
+extends: Packages/CSS/CSS.sublime-syntax
+
+contexts:
+  prototype:
+    - meta_prepend: true
+    - include: php-tag-start
+
+  php-tag-start:
+    - match: <\?(?i:php|=)?(?![^?]*\?>)
+      scope: punctuation.section.embedded.begin.php
+      push:
+        - meta_scope: meta.embedded.block.php
+        - meta_content_scope: source.php.embedded.css
+        - include: php-end-tag-pop
+        - include: scope:source.php
```

Comment:
PHP Source itself, pushes into HTML if it detects the `?>` tag, which results in CSS not being highlighted correctly afterwards. I guess, what we also need is a _PHP (for CSS).sublime-syntax_ which overrides the relevant contexts to make sure we push into CSS and pops back once the next `<?` is matched.

Reply:
Reading the code for this (in PHP Source), shouldn't the match for `?>` be redundant? Because this currently works just fine for me.

![2021-04-29_22-53-49](https://user-images.githubusercontent.com/931051/116617159-d49b9f80-a93d-11eb-9e29-6e3ca51adf57.png)

Reply:
`?>` is not reduntant. See what happens when removing the match from CSS.

![grafik](https://user-images.githubusercontent.com/16542113/116696525-4fa69980-a9c2-11eb-904b-3d794589d168.png)

Your examples work only, because the statements used don't push into any context, thus the `?>` pattern in CSS file recovers state correctly.

As soon as you use a statement which pushes into any context in PHP, it will push into HTML when matching a `?>`, which causes the CSS syntax to break after such a statetement.

![grafik](https://user-images.githubusercontent.com/16542113/116696414-2b4abd00-a9c2-11eb-9ace-8c6e1675d0a3.png)

Fortunately those constructs seem quite rare in PHP.

What might be needed to fix those issues is to override the `embedded-html` context to push into CSS/JS instead of HTML, but I am not sure how good that works with regards to mixing two context sensitive syntaxes.

Reply:
Ah yes, I understand. I forgot that the Php syntax itself tries to maintain a context stack for blocks such as function definitions.

While I agree that these are issues, I don't want to solve them with this PR as they existed before and it is already bigger than I intended